local-volume/provisioner/pkg/common/common.go

```diff
@@ -57,6 +64,8 @@ type MountConfig struct {
 	HostDir string `json:"hostDir"`
 	// The mount point of the hostpath volume
 	MountDir string `json:"mountDir"`
+	// The volumeType "block" or "file". If nil, the type defaults to "file"
+	VolumeType string `json:"volumeType"`
```

Comment:
@msau42 Does that mean a given class of storage can consist of both file PVs and block PVs? It seemed like in last week's discussion the SIG was leaning towards not having volumeType in PVC (or did I misunderstand that?). In that case, wouldn't specifying a PVC with a specific storage class give the POD either a block device or file system, depending on which one is available?